src/ocean/util/log/layout/LayoutSimple.d

```diff
@@ -89,3 +94,21 @@ public class LayoutSimple : Appender.Layout
         return Integer.formatter(tmp, i, 'u', '?', 8);
     }
 }
+
+unittest
+{
+    mstring result = new mstring(2048);
+    result.length = 0;
+    enableStomping(result);
+
+    scope dg = (cstring v) { result ~= v; };
+    scope layout = new LayoutSimple();
+    LogEvent event = {
+        msg_: "Have you met Ted?",
+        name_: "Barney",
+        level_: ILogger.Level.Warn,
+    };
+
+    testNoAlloc(layout.format(event, dg));
+    test!("==")(result, "Warn [Barney] - Have you met Ted?");
```

Comment:
I would think of this more as 'Info' level :)

Reply:
No! It's Warn for sure :).